gsa/src/web/pages/tasks/graphql.js

```diff
@@ -32,6 +34,17 @@ export const useGetCaps = () => {
   return toFruitfulQuery(useQuery)(GET_CAPS);
 };
 
+export const useCaps = caps => {
```

Comment:
Looks good. Just some additions :-)

Could you move it to web/utils/useCapabilities.js and rename the hook to useCapabilities? We have code completion in our editors therefore we should use meaningful and expressive names for function etc.